File: Ej 7 (Inv Cadenas)/Java/InvCadenas.java
//DeathbatO

import java.util.Scanner;

public class InvCadenas {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input;

        System.out.print("Por favor ingrese la cadena a invertir: ");
        input = sc.nextLine();
        char cadena[] = new char[input.length()];
        cadena = input.toCharArray();
        System.out.println("Cadena invertida: ");
        for(int i = input.length()-1; i >= 0; i--){
            System.out.print(cadena[i]);
        }   
        sc.close();
    }
}

File: Ej 7 (Inv Cadenas)/Kotlin/InvCadenas.kt
// DeathbatO

fun main() {
    print("Por favor ingrese la cadena a invertir: ")
    val cadena = readLine() ?: ""
    
    println("Cadena invertida:")
    var k = cadena.length - 1
    while (k >= 0) {
        println(cadena[k])
        k--
    }
}